### PR TITLE
Zhangt6 hopkinton

### LINKED ISSUE #94: zhangt6 hometown MA
# Visit Hopkinton MA

- Boston marathon starting point. 
- EMC
- Restaurant

### PR DESCRIPTION
**Pull request by zhangt6**
added a file zhangt6_ma. This pull request resolves #94.

@revurp @sunw3 test review pls.

### CODE REVIEW

**Comment by zhangt6** [comment]
@sunw3 Thanks.

File: zhangt6-ma.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Hopkinton, MA
+
+### Great places to eat
```
Comment (jayeshsaita) [inline zhangt6-ma.md]: Hey hi

Comment (jayeshsaita) [inline zhangt6-ma.md]: wassup

**Comment by zhangt6** [comment]
@jayeshsaita Hey, could you please approve the request?

**Pull request merged**